### DISCUSSION TITLE
TOILETTEN

### DISCUSSION THREAD

**Discussion by Symerzen** [nzp-team/nzportable#739] [Map Releases]
![TOILETTEN](https://github.com/nzp-team/nzportable/assets/170592051/fc12b994-2670-4af5-aaf8-68b2fed460cc)

A map made in a few days to practice with map making.

Comes with short 2 part Easter egg and buyable ending.

Should be fine in Co-op, tested with 1-2 people but 4 should be possible.

# [DOWNLOAD](https://archive.org/download/toiletten_202405/TOILETTEN.zip)

**Comment by LegendDD44** [comment]
beat the first step easy but cant figure out the second

**Reply by LegendDD44** [reply]
nevermind i did it

**Reply by IFERNANDO115** [reply]
that you really know how to unlock it the pack a punch

**Comment by nonMat** [comment]
WOW this was surprisingly fun!!!
Had a lot of fun with the Easter Egg!!!

**Comment by DavidCojocaru04** [comment]
This was one of my most favourite small challenge maps but unfortunately the download link no longer works.. :(((

**Reply by Symerzen** [reply]
The link is hosted on the Internet Archives, but right now their site seems to be having some connection issues. Try again tomorrow perhaps, or whenever the site becomes available again.

**Reply by DavidCojocaru04** [reply]
Ah ok, thank you for letting me know, I'll try again tomorrow then! ^^

**Reply by M0THRAK** [reply]
Hey, you can use this download for the time being: https://www.mediafire.com/file/o7tbihvh5gw1eks/TOILETTEN_Archive.zip/file
